services/logging/src/main/decorators/Trace.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { PlatformController } from '../common/platform.controller'
+
+// TODO: incorporate messaging into this and make it more general purpose
+
+function Trace(): MethodDecorator {
```

Comment:
I don't see an `@trace` usage anywhere, is nestjs using this implicitly?

Reply:
No, nobody is using it at the moment. It's something I know we'll want to use in other services, so I wanted to keep the code around since it took a bit of research to figure out how to get it working properly, but at the moment it's not being used anywhere. Eventually we should move it to a common place along with the PlatformController, Validate, and PlatformMicroservice.